data/Govamarket.jason

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Clinton Chidera",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend developer",
+  "avatar": https://github.com/Govamarket.png"
```

Comment:
Currently, you don't need to add an avatar. It will automatically fetch from your GitHub.
``` suggestion
 
```